src/test/groovy/pl/futurecollars/invoicing/db/FileBasedDatabaseTest.groovy

```diff
@@ -1,16 +1,19 @@
 package pl.futurecollars.invoicing.db
 
-import pl.futurecollars.invoicing.model.Invoice
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
 import pl.futurecollars.invoicing.utils.FileService
-import pl.futurecollars.invoicing.utils.JsonService
 
+@SpringBootTest
 class FileBasedDatabaseTest extends DatabaseTest {
+
+    @Autowired
+    FileBasedDatabase fileBasedDatabase
+
     @Override
     Database getDatabaseInstance() {
         def fileService = new FileService()
         fileService.clearDatabase()
```

Comment:
Not sure if this would work as this FileService wasn't passed to fileBasedDatabase constructor